### PR TITLE
SG-22787: Shotgun to ShotGrid

### PR DESCRIPTION
**Pull request by jfboismenu**
Rebranded, and fixed a few warnings to make the output of the doc generation process a little cleaner.

### CODE REVIEW

File: shotgun_api3/shotgun.py [hunk 1]
```diff
@@ -774,9 +774,6 @@ def info(self):
         >>> sg.info()
         {'full_version': [8, 2, 1, 0], 'version': [8, 2, 1], 'user_authentication_method': 'default', ...}
 
-        Tokens and values
```
Comment (jfboismenu) [inline shotgun_api3/shotgun.py:777]: This was never rendered out to html and generated warnings. It made sense to remove as the doc is prefectly readable without it.

File: shotgun_api3/shotgun.py [hunk 2]
```diff
@@ -2730,13 +2727,11 @@ def get_attachment_download_url(self, attachment):
         .. note::
             Support for passing in an int representing the Attachment ``id`` is deprecated
 
-        .. todo::
```
Comment (jfboismenu) [inline shotgun_api3/shotgun.py:2733]: `todo` is not a valid token in sphinx.

File: .travis.yml [hunk 3]
```diff
@@ -17,26 +17,13 @@ language: python
 matrix:
   include:
 
-    # Use trusty dist for python 2.6, since the built-in xenial distro (default)
-    # does not support Python 2.6
-    # See https://docs.travis-ci.com/user/reference/xenial/#python-support
-
-    # Python 2.6, Server #1
```
Comment (jfboismenu) [inline .travis.yml:24]: Python 2.6 was deprecated on May 1rst and the build is failing, so let's get rid of it.

File: HISTORY.rst [hunk 4]
```diff
@@ -37,61 +37,61 @@ v3.1.2 (2019 Sept 17)
   methods ``schema_entity_read()``, ``schema_field_read()``, and ``schema_read()``.
 
 v3.1.1 (2019 August 29)
-=====================
+=======================
```
Comment (jfboismenu) [inline HISTORY.rst:40]: This fixes a LOT of warnings.

File: README.md [hunk 5]
```diff
@@ -1,36 +1,36 @@
...
-* [Installing](http://developer.shotgunsoftware.com/python-api/installation.html)
-* [Tutorials](http://developer.shotgunsoftware.com/python-api/cookbook/tutorials.html)
-* [API Reference](http://developer.shotgunsoftware.com/python-api/reference.html)
-* [Data Types](http://developer.shotgunsoftware.com/python-api/reference.html#data-types)
-* [Filter Syntax](http://developer.shotgunsoftware.com/python-api/reference.html#filter-syntax)
+* [Installing](http://developer.shotgridsoftware.com/python-api/installation.html)
+* [Tutorials](http://developer.shotgridsoftware.com/python-api/cookbook/tutorials.html)
+* [API Reference](http://developer.shotgridsoftware.com/python-api/reference.html)
+* [Data Types](http://developer.shotgridsoftware.com/python-api/reference.html#data-types)
+* [Filter Syntax](http://developer.shotgridsoftware.com/python-api/reference.html#filter-syntax)
```
Comment (eshokrgozar) [inline README.md:29]: These links don't work right now. Are we sure they are going to be online when this is released?

**Review by eshokrgozar — APPROVED** [review]
LGTM as long as we're okay with the broken links for now.

**Pull request merged**